Update config-settings.rst (#3428)
mattermost/docs: source/administration/config-settings.rst

.. warning:: Verify server memory can support your setting choice. Large file sizes increase the risk of server crashes and failed uploads due to network disruptions.

> .. note:: If you use a proxy or load balancer in front of Mattermost make sure to also adjust the settings for that as well. When running NGINX this would be accomplished using ``client_max_body_size`` and for Apache using ``LimitRequestBody``.

Reviewer: ```suggestion
.. note:: 
If you use a proxy or load balancer in front of Mattermost its settings need to be adjusted accordingly. For NGINX use ``client_max_body_size``. For Apache use ``LimitRequestBody``.
```
